ballerina/types.bal

```diff
@@ -23,19 +25,17 @@ public type QueueManagerConfiguration record {|
     string password?;
 |};
 
-public enum ConnectionOpenOptions {
-    MQOO_OUTPUT = "MQOO_OUTPUT",
-    MQOO_INPUT_AS_Q_DEF = "MQOO_INPUT_AS_Q_DEF",
-    MQOO_INPUT_EXCLUSIVE = "MQOO_INPUT_EXCLUSIVE",
-    MQOO_INPUT_SHARED = "MQOO_INPUT_SHARED"
-}
+public type GetMessageOptions record {|
```

Comment:
I am trying to understand the content of this record. We have a `GetMessageOptions` record, and that record has a `gmOptions` int field.  I assume `gmOptions` is the same as `getMessageOptions`.  This is kinda confusing :) 

Do we need this record type? Can't we simply add function parameters?

Reply:
There are some more options that we have not added at the moment which is there in the java api - https://www.ibm.com/docs/en/ibm-mq/9.3?topic=java-mqgetmessageoptions. However, will rethink about the field naming

Reply:
Updated the variable name in https://github.com/ballerina-platform/module-ballerinax-ibm.ibmmq/pull/5/commits/cbcc5a239bb4d93bb94a39cada814822a15e1aa1